src/main/java/com/github/vkorobkov/jfixtures/fluent/impl/JFixturesResultBase.java

```diff
@@ -0,0 +1,34 @@
+package com.github.vkorobkov.jfixtures.fluent.impl;
+
+import com.github.vkorobkov.jfixtures.config.ConfigLoader;
+import com.github.vkorobkov.jfixtures.fluent.JFixturesResult;
+import com.github.vkorobkov.jfixtures.instructions.CleanTable;
+import com.github.vkorobkov.jfixtures.instructions.CustomSql;
+import com.github.vkorobkov.jfixtures.instructions.InsertRow;
+import com.github.vkorobkov.jfixtures.instructions.Instruction;
+import com.github.vkorobkov.jfixtures.loader.FixturesLoader;
+import com.github.vkorobkov.jfixtures.processor.Processor;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.val;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElements;
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+abstract class JFixturesResultBase implements JFixturesResult {
+    private String fixturesFolder;
+
+    @XmlElements({
```

Comment:
Can we move this XML specific annotation to `XmlJFixturesResultImpl` since the base class _should not_ know about the implementation and technical details of its children?
```java
@Override
@XmlElements( .... )
List<Instruction> getInstructions() {
  return super.getInstructions();
}
```

Reply:
:100: